homeassistant/components/light/tellstick.py

```diff
@@ -5,126 +5,84 @@
 https://home-assistant.io/components/light.tellstick/
 """
 from homeassistant.components.light import ATTR_BRIGHTNESS, Light
-from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-
+from homeassistant.components.tellstick import (TellstickDevice,
+                                                TELLCORE_REGISTRY,
+                                                DISCOVER_DEVICES,
+                                                DISCOVER_CONFIG)
 REQUIREMENTS = ['tellcore-py==1.1.2']
```

Comment:
Shouldn't this requirement now be part of the main component and no longer be part of the platform? Also, does this platform have a dependency on the main component?